readme.md

```diff
@@ -66,7 +65,7 @@ One of the lines of code in the following snippet will throw an error. Which one
 /*8*/ }
 ```
 
-> Your answer...
+> Line 3 will throw an error. `use strict` prevents you from using any variables without declaring them first with `var`. `i` hasn't been declared.
 
```

Comment:
My thoughts are that we should replace this question with a a larger topic. `"use strict";` has only come up in Angular and even then it's not essential.

Maybe a Rails question? At the moment there's only one Ruby question.